### DISCUSSION TITLE
Error: git add .

### DISCUSSION THREAD

**Discussion by yasharyan0904** [Cyfrin/foundry-full-course-cu#1786] [Q&A]
I am getting the following error:
Can anyone help me out!!
<img width="870" alt="Capture__S" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/144695005/267cf138-fede-4686-b14a-e5de8dc0d7d3">

**Comment by urosognjenovic** [comment]
Is this the output you're getting when you run ```git status```?

**Comment by EngrPips** [comment]
Hello @yasharyan0904, I don't think that output is an error; instead, it is information about which files you have tracked and which ones you have yet to track.
